salt_resources/states/basebox/miniconda.sls

```diff
@@ -0,0 +1,9 @@
+download_miniconda_installer:
+  cmd.run:
+    - name: wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O /home/vagrant/miniconda.sh
+    - unless: test -f /usr/local/bin/miniconda/bin/conda
+
+ensure_miniconda_is_installed:
+  cmd.run:
+    - name: bash /home/vagrant/miniconda.sh -b -p /usr/local/bin/miniconda
```

Comment:
I thought miniconda modified your bashrc for you. Though, it's not mentioned on their silent installation page https://conda.io/docs/help/silent.html, so I guess it doesn't do that when silently installed.

If we add it to the deescalated user's bashrc, then that bash code would be ran when you log in to that user. I don't see a need atm to reload the bash profile / rc as Salt is executing it's states.

About your link, I'm not surprised vagrant broke and then fixed something again. I haven't ran into that particular prompt issue though.